Formula/openai-whisper.rb

```diff
@@ -39,35 +40,41 @@ class OpenaiWhisper < Formula
     sha256 "34a17436ed1e96697a86f9de3d15a3b0be01d8bc8de9c1dffd59fb8234ed5307"
   end
 
-  resource "more-itertools" do
-    url "https://files.pythonhosted.org/packages/13/b3/397aa9668da8b1f0c307bc474608653d46122ae0563d1d32f60e24fa0cbd/more-itertools-9.0.0.tar.gz"
-    sha256 "5a6257e40878ef0520b1803990e3e22303a41b5714006c32a3fd8304b26ea1ab"
+  resource "llvmlite" do
```

Comment:
do you think we should have this as separate formula?

Reply:
Why? What's special about it, that it should be a separate formula? (I'm not discounting the idea, just curious.)

Reply:
nothing special (just curious), we do intend to extract resource into formula if possible. (but this can be totally done at a different time)

Reply:
It may make sense to provide `numba` as a separate formula, just like there are existing formulae for `numpy`, `scipy`, and `cython`. As I understand it, `llvmlite` is a dependency that is primarily used by `numba`, although it could in the future be used by other projects as well.

But since this is literally the first formula in `homebrew-core` that relies on `numba`, I figured it made sense to just leave it as a resource for now. It can be factored out into a formula if/when other formulae are introduced that can use it.